content/bn/docs/concepts/overview/_index.md

```diff
@@ -0,0 +1,178 @@
+---
+
+title: "ওভারভিউ"
+description: >
+  কুবারনেটিস হল একটি পোর্টেবল, বর্ধনশীল, ওপেন সোর্স প্ল্যাটফর্ম যা কনটেইনারাইজড ওয়ার্কলোড এবং পরিষেবাগুলি পরিচালনা করার জন্য, ঘোষণামূলক কনফিগারেশন এবং অটোমেশন উভয়কেই সহজতর করে। এটির একটি বড়, দ্রুত বর্ধনশীল ইকোসিস্টেম রয়েছে। কুবারনেটিস পরিষেবাগুলি, সমর্থন, এবং সরঞ্জাম ব্যাপকভাবে সহজলভ্য।
+content_type: concept
+weight: 20
+card:
+  name: concepts
+  weight: 10
+  anchors:
+  - anchor: "#why-you-need-kubernetes-and-what-can-it-do"
+    title: কুবারনেটিস কি?
+no_list: true
+---
+
+<!-- overview -->
+এই পৃষ্ঠাটি কুবারনেটিসের একটি পরিপূর্ণ ধারণা প্রদান করে ।
+
+
+<!-- body -->
+
+কুবারনেটিস হল একটি পোর্টেবল, বর্ধনশীল, ওপেন সোর্স প্ল্যাটফর্ম যা কনটেইনারাইজড 
+ওয়ার্কলোড এবং পরিষেবাগুলি পরিচালনা করার জন্য, ঘোষণামূলক কনফিগারেশন এবং অটোমেশন উভয়কেই সহজতর করে। 
+এটির একটি বড়, দ্রুত বর্ধনশীল ইকোসিস্টেম রয়েছে। কুবারনেটিস পরিষেবাগুলি, সমর্থন, এবং সরঞ্জাম ব্যাপকভাবে সহজলভ্য।
+
+কুবারনেটিস নামটি গ্রীক থেকে এসেছে, যার অর্থ হেলমসম্যান বা পাইলট। 
+"K" এবং "s" এর মধ্যে আটটি অক্ষর গণনা করার ফলে একটি সংক্ষিপ্ত রূপ K8s। গুগল ২০১৪ সালে কুবারনেটিস প্রজেক্টটি ওপেন সোর্স করেছে। কুবারনেটিস 
+[Google-এর 15 বছরেরও বেশি অভিজ্ঞতা](/blog/2015/04/borg-predecessor-to-kubernetes/) সম্প্রদায়ের সেরা-প্রজন্মের ধারণা 
+এবং অনুশীলনের সাথে স্কেলে উৎপাদন কাজের চাপগুলিকে একত্রিত করে।
```

Comment:
please break this part into 5 line
![image](https://github.com/kubernetes/website/assets/155321064/48cbf10a-ff64-4e44-87a7-a39b7ed9d44d)